lms/tasks/email_digests.py

```diff
@@ -121,7 +121,7 @@ def send_instructor_email_digest_tasks():
     rate_limit="10/m",
 )
 def send_instructor_email_digest(
-    *, h_userid: str, created_before: str, created_after: Optional[str] = None, **kwargs
+    *, h_userid: str, created_before: str, created_after: str, **kwargs
```

Comment:
`send_instructor_email_digest()` is called from two places: from `send_instructor_email_digest_tasks()` above and from the admin pages. Both now pass the `created_after` argument, so it no longer needs to be optional.